README.md

```diff
@@ -103,6 +103,19 @@ Hutch::Logging.logger = Rails.logger
 See this [RabbitMQ tutorial on topic exchanges](http://www.rabbitmq.com/tutorials/tutorial-five-ruby.html)
 to learn more.
 
+### Message Processing Tracers
+
+Tracers allow you to track message processing. You can set them using `tracer` method.
+
+#### NewRelic
+```ruby
+class FailedPaymentConsumer
+  include Hutch::Consumer
+  consume 'gc.ps.payment.failed'
+  tracer 'NewRelic'
```

Comment:
Ah, just realised that this example is in fact not true as the implementation allows you to set the tracer only via a config. My mistake, sorry. I initially planned to use a consumer method for this but then thought there is no practical value in that and decided it would be more convenient to set it "globally" for all consumers via the config. What do you think?